workers/unity/Assets/Playground/Scripts/UI/ConnectionScreenController.cs

```diff
@@ -27,17 +28,20 @@ public void Awake()
             if (!string.IsNullOrEmpty(hostIp))
             {
                 ipAddressInput.text = hostIp;
-                TryConnect();
```

Comment:
Is this intentional change? If I understand correctly this was here to automatically connect to host locally when we launch with command line arguments

Reply:
This is intentional, should have probably commented that earlier. Now that we can either connect to a local or a cloud deployment, we don‘t want to try and immediately connect to a local deployment all the time. Sure, we could remove the ip in the config, but I think this is a nicer solution as the other one would mean that you would have to look up for local ip again

Reply:
Fine with me if this is explicit change.
I'd point out that this was only triggered from launch menu, where I'd expect people mostly launch for local, but I understand the reasoning.
I think this is another pointer that we need to rethink how we implement client launch configuration so that it is more transparent and processing logic is easier to separate out.

Reply:
I'm not too sure about this. As it is not required to upload the android clients to the cloud deployment, I could imagine that people want to quickly iterate and test changes and try them out in a running cloud deployment.  I simply don't think we should try to immediately connect, if there are now two ways to connect, but this is all code in the playground so developers will likely use a different interface to decide how to connect.

Not sure I understand what you mean by the second point. Do you mean how we decide whether to connect to a local or a cloud deployment?

Reply:
Agree, this approach is more flexible.
For my second point I have a general feeling that we can improve the launch configuration and processing both in tooling and in Playground as a sample, but I don't have concrete suggestions as of the moment. Will try to think more about this and maybe come up with something more concrete.